.circleci/config.yml

```diff
@@ -255,6 +255,115 @@ jobs:
             MOCHA_FILE: integration-tests/test-results/junit/test-results-[hash].xml
           when: always
       - upload_artifacts
+  accessibility_test:
+    resource_class: medium+
```

Comment:
for knowledge, did you try this with `medium`? 
in the insights tab, this doesn't seem to overly stress the instance that much

Reply:
Yes I did, it would sometimes work and another times fail.